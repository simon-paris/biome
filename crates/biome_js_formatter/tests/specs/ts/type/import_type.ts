type ImportType1 = typeof import('source');

type ImportType2 = import('source');

type QualifiedImportType = typeof import('source').Qualified<TypeParams>;

type WithTypeAssertion = import("a.json", { assert: { type: "json" } });

const ShouldNotHaveTrailingCommas = require("long/long/long/long/long/long/long/long/long/long/path.json") as typeof import("/long/long/long/long/long/long/long/long/long/long/path.json");
