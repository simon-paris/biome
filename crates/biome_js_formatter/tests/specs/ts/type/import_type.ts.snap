---
source: crates/biome_formatter_test/src/snapshot_builder.rs
---
# Input

```ts
type ImportType1 = typeof import('source');

type ImportType2 = import('source');

type QualifiedImportType = typeof import('source').Qualified<TypeParams>;

type WithTypeAssertion = import("a.json", { assert: { type: "json" } });

const ShouldNotHaveTrailingCommas = require("long/long/long/long/long/long/long/long/long/long/path.json") as typeof import("/long/long/long/long/long/long/long/long/long/long/path.json");

```


=============================

# Outputs

## Output 1

-----
Indent style: Tab
Indent width: 2
Line ending: LF
Line width: 80
Quote style: Double Quotes
JSX quote style: Double Quotes
Quote properties: As needed
Trailing commas: All
Semicolons: Always
Arrow parentheses: Always
Bracket spacing: true
Bracket same line: false
Attribute Position: Auto
-----

```ts
type ImportType1 = typeof import("source");

type ImportType2 = import("source");

type QualifiedImportType = typeof import("source").Qualified<TypeParams>;

type WithTypeAssertion = import("a.json", { assert: { type: "json" } } );

const ShouldNotHaveTrailingCommas =
	require("long/long/long/long/long/long/long/long/long/long/path.json") as typeof import("/long/long/long/long/long/long/long/long/long/long/path.json");
```

# Lines exceeding max width of 80 characters
```
   10: 	require("long/long/long/long/long/long/long/long/long/long/path.json") as typeof import("/long/long/long/long/long/long/long/long/long/long/path.json");
```
